ledger-core/virtual/execute/execute.go

```diff
@@ -64,9 +66,10 @@ type SMExecute struct {
 	pulseSlot             *conveyor.PulseSlot
 	authenticationService authentication.Service
 
-	outgoing        *payload.VCallRequest
-	outgoingObject  reference.Global
-	outgoingWasSent bool
+	outgoing             *payload.VCallRequest
+	outgoingObject       reference.Global
+	outgoingWasSent      bool
+	retryOutgoingCounter int
```

Comment:
outgoingWasSent is duplicated by retryOutgoingCounter

Reply:
done